index.bs

```diff
@@ -187,7 +188,27 @@ Presentation API Requirements {#requirements-presentation-api}
 Remote Playback API Requirements {#requirements-remote-playback}
 ----------------------------------------------------------------
 
-Issue(3): Requirements for Remote Playback API.
+1.  A [=controlling user agent=] must be able to find out whether there is at
+    least one compatible [=remote playback device=] available for a given
+    {{HTMLMediaElement}}, both instantaneously and continuously.
+
+2.  A controlling user agent must be able to start sending the content of an
+    {{HTMLMediaElement}} to a compatible remote playback device.
+
+3.  The {{HTMLMediaElement}} content to be sent should include both media data
+    and text tracks.
+
+4.  During remote playback, the controlling user agent and the remote playback
```

Comment:
Should we define what controlling means here?

Reply:
"controlling user agent" is a term defined in this spec, but inherited from the Presentation API.   The Remote Playback requirements use the generic term "user agent" which I can also use here if you prefer.

We have a little work to do to reconcile the terminology between the three specs (Issue #144) :-)